src/Config/ValueProvider/TestFrameworkConfigPathProvider.php

```diff
@@ -93,7 +93,7 @@ private function getValidator(string $testFramework): \Closure
         };
     }
 
-    private function askTestFrameworkConfigLocation(InputInterface $input, OutputInterface $output, array $dirsInCurrentDir, string $testFramework, $defaultValue): string
+    private function askTestFrameworkConfigLocation(InputInterface $input, OutputInterface $output, array $dirsInCurrentDir, string $testFramework, string $defaultValue): string
```

Comment:
what about to add default value ?

Reply:
It is set explicitly on lines 53 and 57 above